kinto/config/kinto.tpl

```diff
@@ -59,6 +160,60 @@ kinto.includes = kinto.plugins.default_bucket
 # fxa-oauth.webapp.authorized_domains = *
 
 #
+# Plugins
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#plugin
+kinto.includes = kinto.plugins.default_bucket
+
+#
+# Notifications
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#notifications
+#
+# 
+# Filtering
+#
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#filtering
```

Comment:
This filtering section may be a bit confusing. A very simple example of notification can be useful though (see maybe https://github.com/Kinto/kinto-changes/blob/master/README.rst)